community/sway/etc/sway/config.d/99-autostart-applications.conf

```
@@ -18,4 +18,5 @@ exec_always {
     # restart kanshi https://github.com/emersion/kanshi/issues/43#issuecomment-531679213
     '[ -x "$(command -v kanshi)" ] && pkill kanshi; exec kanshi'
     '[ -x "$(command -v sworkstyle)" ] && pkill sworkstyle; sworkstyle &> /tmp/sworkstyle.log'
+    '[ -x "$(command -v playerctl)" ] && pkill playerctl; playerctl -a metadata --format '' --follow | while read line; do pkill -RTMIN+5 waybar; done'
```

```suggestion
    '[ -x "$(command -v playerctl)" ] && pkill playerctl; playerctl -a metadata --format '{{status}} {{title}}' --follow | while read line; do pkill -RTMIN+5 waybar; done'
```
Without those it won't trigger a newline, and thus won't trigger a reload